src/modules/tests/test_data/test-data.reducer.ts

```diff
@@ -269,6 +272,7 @@ export function testDataReducer(
       return {
         ...state,
         vehicleChecks: {
+          ...state.vehicleChecks,
           tellMeQuestion: {
             ...state.vehicleChecks.tellMeQuestion,
             outcome: 'P',
```

Comment:
Could this come from constants like `CompetencyOutcome` ?

Reply:
Yes, that would be good. We don't yet have it in the enum, but I can add it.

Reply:
Added the `CompetencyOutcome.P`